data/json/items/gun/223.json

```diff
@@ -496,25 +496,25 @@
     "melee_damage": { "bash": 12 }
   },
   {
-    "//": "Name changed to match the non-branded version. Also changed to the SIG 553, as the 552 is discontinued and the 553 is still in production and is identical to the 552.",
+    "//": "Changed from SG 553, which itself was originally the Sig 550, to the civilian legal copy",
     "id": "sig_assault_rifle",
-    "copy-from": "rifle_auto",
+    "copy-from": "rifle_semi",
     "looks_like": "modular_ar15",
     "type": "GUN",
-    "name": { "str": "SIG assault rifle" },
-    "description": "A compact selective fire automatic rifle designed for the Swiss military.  It features a three-round burst mode and an integrated folding stock.  It is chambered in 5.56x45mm and accepts STANAG magazines.",
-    "weight": "3200 g",
-    "volume": "2950 ml",
-    "longest_side": "741 mm",
-    "barrel_length": "348 mm",
+    "name": { "str": "SIG civilian rifle" },
+    "description": "A civilian clone of an automatic swiss rifle for military service.  Accepts STANAG magazines.",
```

Comment:
```suggestion
    "description": "A civilian variant of an automatic Swiss rifle for military service.  Accepts STANAG magazines.",
```

Reply:
Ah whoops, had a fix committed but forgot to push